### DISCUSSION TITLE
osu! minimizing to windows tray when pressing spacebar

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#18305] [Q&A] [answered]
So whenever I press the spacebar on osu! it minimizes it to the windows tray. It doesn't matter where on osu! it can be. In a song, title screen. It does it everytime. I have tried it on and off in full screen. Tried it with compatibility mode on. Tried it with Cutting Edge and Stable releases. Also made sure it wasn't my keyboard by using the on-screen keyboard and pressing the spacebar on that. Still the same issue. Here is a video of this happening: https://streamable.com/x69wae

Don't really know what else to try so any suggestions would be great.

My PC specs:
![image](https://user-images.githubusercontent.com/105034762/168625847-24aa50bb-c69b-4644-91a8-3a41d244ac3e.png)
![image](https://user-images.githubusercontent.com/105034762/168625902-c3cea684-c408-4b55-9d50-dd74904b5e95.png)

**Comment by bdach** [comment, marked as answer]
Stable issue reports go here: https://github.com/ppy/osu-stable-issues

**Reply by ghost** [reply]
alright. i will make another discussion now.
